client/payment-methods/index.js

```diff
@@ -65,9 +65,11 @@ const PaymentMethods = () => {
 		updateEnabledPaymentMethodIds: updateEnabledMethodIds,
 	} = useEnabledPaymentMethodIds();
 
-	const enabledMethods = enabledMethodIds.map( ( methodId ) =>
-		availableMethods.find( ( method ) => method.id === methodId )
-	);
+	const enabledMethods = enabledMethodIds
+		.map( ( methodId ) =>
+			availableMethods.find( ( method ) => method.id === methodId )
+		)
+		.filter( ( method ) => method !== undefined );
```

Comment:
Once https://github.com/Automattic/woocommerce-payments/pull/1848 is merged, I don't think we'll need `.filter` anymore.
Also worth mentioning a little cool shortcut ;)
```suggestion
		.filter( Boolean );
```

Reply:
#1848 is merged. I removed the filter after merging latest `develop`.